Version check

PabloG6 opened the pull request:
Completed the version check requirements for the Codeu Chat server.

Commit:
Committing completed version check

markpilloff commented on src/codeu/chat/client/core/View.java:
```diff
@@ -14,21 +14,18 @@
 import codeu.chat.util.Uuid;
 import codeu.chat.util.connections.Connection;
 import codeu.chat.util.connections.ConnectionSource;
+import sun.nio.ch.Net;
```
I don't think you need this? I don't see this referenced elsewhere in your code.

markpilloff commented on src/codeu/chat/ServerMain.java:
```diff
@@ -76,7 +76,7 @@ public static void main(String[] args) {
     ) {
 
-      LOG.info("Starting server...");
+      LOG.info("Starting o...");
```
unwanted change?

markpilloff commented on src/codeu/chat/client/core/View.java:
```diff
@@ -92,6 +89,26 @@ public View(ConnectionSource source) {
+        final Uuid version = Uuid.SERIALIZER.read(connection.in());
+        return new ServerInfo(version);
+      } else {
+        System.out.println("Oops, something wen't wrong. Please try again");
```
A more descriptive message would be helpful. E.g., "Expected SERVER_INFO_RESPONSE but didn't receive it."

markpilloff commented on .gitignore:
```diff
@@ -1,2 +1,4 @@
 bin/
 *.log
+.idea/
```
this change probably belongs on your local system but not as part of the group project

markpilloff reviewed (changes requested):
Nice job! 
Mostly minor comments. Also, you've included an empty file: chat_server_log.log.lck

Commit:
Committing changes to meet version check activity standard.

Commit:
Registered info command

Commit:
Committing addition of getServerInfo interface method to View.java

Commit:
Committing addition of getServerInfo method from BasicView.jAVA interface to View.java

Commit:
Committing minor changes

kimberlylo committed:
Removed sun.nio.ch.Net, replaced error message

Removed sun.nio.ch.Net from imports, replaced error message with more
descriptive message about not receiving SERVER_INFO_RESPONSE

kimberlylo committed:
Revert "Removed sun.nio.ch.Net, replaced error message"

This reverts commit 436b4b76888253c1b305343ea305a768011362e9.

kimberlylo committed:
Removed sun.nio.ch.Net, changed error message

Removed sun.nio.ch.Net from imports, changed error message with more
descriptive one describing no SERVER_INFO_RESPONSE received

kimberlylo committed:
Update .gitignore

kimberlylo committed:
Delete chat_server_log.log.lck

kimberlylo committed:
Merge branch 'master' into version-check
